src/entity/Comment.ts

```diff
@@ -45,6 +49,14 @@ export class Comment {
   @Column({ default: false })
   featured: boolean;
 
+  // @Column({ type: 'jsonb', default: {} })
+  // @Index('IDX_user_flags_banned', { synchronize: false })
+  // flags: CommentFlags;
```

Comment:
Why not flags?

Reply:
[See other comment](https://github.com/dailydotdev/daily-api/pull/2089#discussion_r1694993079)

Reply:
Refactored to use flags